WORKSPACE

```diff
@@ -1,11 +1,25 @@
-workspace(name = "io_bazel_rules_play_routes")
+workspace(name = "compiler_cli")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-rules_scala_annex_version = "eeefae00b6e12ffd514f49c399f8d0abff7e4182" # update this as needed
+# rules_jvm_external
+RULES_JVM_EXTERNAL_TAG = "2.5"
+http_archive(
+    name = "rules_jvm_external",
+    sha256 = "249e8129914be6d987ca57754516be35a14ea866c616041ff0cd32ea94d2f3a1",
+    strip_prefix = "rules_jvm_external-{}".format(RULES_JVM_EXTERNAL_TAG),
+    type = "zip",
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/{}.zip".format(RULES_JVM_EXTERNAL_TAG),
+)
+
+load("//:workspace.bzl", "play_routes_repositories")
+play_routes_repositories()
```

Comment:
Why are we not using pinned json?

Reply:
There aren't very many dependencies, and other Bazel projects shouldn't be depending on this workspace.

Reply:
> other Bazel projects shouldn't be depending on this workspace

Agree. While if it helps speed up CI, it may still be worth it.

Reply:
Good point. I'll add it.